### PR TITLE
Enable Proto Actor Tracing for TraceLens

### PR DESCRIPTION
**Pull request by sfmskywalker**
This PR turns on tracing for the WorkflowInstance actor and provides a sample configuration for OpenTelemetry configured to send signals to TraceLens as the collector.

This change is [Reviewable](https://reviewable.io/reviews/elsa-workflows/elsa-core/5800)

### CODE REVIEW

File: src/bundles/Elsa.Server.Web/Elsa.Server.Web.csproj [hunk 1]
```diff
@@ -49,13 +49,21 @@
...
+        <PackageReference Include="OpenTelemetry.Exporter.OpenTelemetryProtocol"/>
+        <PackageReference Include="OpenTelemetry.Instrumentation.StackExchangeRedis"/>
     </ItemGroup>
     
     <!--Overridden for vulnaribility reasons with dependencies referencing older versions.-->
     <ItemGroup>
         <PackageReference Include="System.Text.Json" VersionOverride="8.0.4" />
     </ItemGroup>
+    <ItemGroup>
+        <Folder Include="App_Data\"/>
```
Comment (raymonddenhaan) [inline src/bundles/Elsa.Server.Web/Elsa.Server.Web.csproj:66]: Do we need the empty folder?

File: src/modules/Elsa.ProtoActor/Features/ProtoActorFeature.cs [hunk 2]
```diff
@@ -85,14 +78,17 @@ public override void Apply()
...
 
-            var clusterProvider = ClusterProvider(sp);
-            var system = new ActorSystem(systemConfig).WithServiceProvider(sp);
-            var workflowGrainProps = system.DI().PropsFor<WorkflowInstanceActor>();
+            ConfigureActorSystemConfig(sp, actorSystemConfig);
+
+            var clusterProvider = CreateClusterProvider(sp);
+            var system = new ActorSystem(actorSystemConfig).WithServiceProvider(sp);
+
+            var workflowGrainProps = system.DI().PropsFor<WorkflowInstanceActor>().WithTracing();
```
Comment (raymonddenhaan) [inline src/modules/Elsa.ProtoActor/Features/ProtoActorFeature.cs:91]: Do we always want tracing enabled, or make this configurable?

**Review by raymonddenhaan — APPROVED** [review]

**Comment by raymonddenhaan** [comment]
Please address the build failure as well

File: src/modules/Elsa.ProtoActor/Features/ProtoActorFeature.cs (hunk 2 above)
Comment (sfmskywalker) [reply src/modules/Elsa.ProtoActor/Features/ProtoActorFeature.cs:91]: Fair question - not everyone may be interested in the overhead of sending OpenTelemetry signals. I'll make it configurable.

**Pull request merged**